clang/lib/Driver/OffloadBundler.cpp

```diff
@@ -687,12 +687,12 @@ class ObjectFileHandler final : public FileHandler {
         if (Error Err = Symbol.printName(NameOS))
           return std::move(Err);
 
-        // If we are dealing with a bitcode file do not add special globals
-        // llvm.used and llvm.compiler.used and __AsanDeviceGlobalMetadata to
+        // If we are dealing with a bitcode file do not add special globals to
         // the list of defined symbols.
         if (SF->isIR() &&
             (Name == "llvm.used" || Name == "llvm.compiler.used" ||
-             Name == "__AsanDeviceGlobalMetadata"))
+             Name == "__AsanDeviceGlobalMetadata" ||
+             Name == "__MsanKernelMetadata"))
```

Comment:
Why the name is different for ASAN and MSAN?

Reply:
Good idea. I will merge them in other patch.

Reply:
Currently, I handle them all in SanitizerKernelMetadata Pass, and don't use a same name, because I'm not sure if their content will be different.